sympy/solvers/ode/tests/test_systems.py

```diff
@@ -1273,7 +1269,18 @@ def test_sysode_linear_neq_order1_type2():
     assert dsolve(eq26) == sol26
     assert checksysodesol(eq26 , sol26) == (True , [0,0])
 
+    # Test Case added for issue #22715
+    # https://github.com/sympy/sympy/issues/22715
 
+    eqn1 = Eq(diff(x(t),t),-1*y(t)+10)
+    eqn2 = Eq(diff(y(t),t),5*x(t)-2*y(t)+3)
+    eq27 = [eqn1, eqn2]
+    sol27 = [Eq(x(t), (C1/5 - 2*C2/5)*exp(-t)*cos(2*t) -
+         (2*C1/5 + C2/5)*exp(-t)*sin(2*t) + 17*sin(2*t)**2/5 + 17*cos(2*t)**2/5),
+          Eq(y(t), C1*exp(-t)*cos(2*t) - C2*exp(-t)*sin(2*t) + 10*sin(2*t)**2 +
+           10*cos(2*t)**2)]
```

Comment:
Try to format this nicely with indentation that makes it easier to see what this solution actually is when looking at the code. Something like:
```python
    sol27 = [Eq(x(t), (C1/5 - 2*C2/5)*exp(-t)*cos(2*t)
                    - (2*C1/5 + C2/5)*exp(-t)*sin(2*t)
                    + 17*sin(2*t)**2/5 + 17*cos(2*t)**2/5),
             Eq(y(t), C1*exp(-t)*cos(2*t)
                    - C2*exp(-t)*sin(2*t)
                    + 10*sin(2*t)**2 + 10*cos(2*t)**2)]
```